src/devices/realsense2/realsense2Driver.h

```diff
@@ -0,0 +1,226 @@
+﻿/*
+ * Copyright (C) 2006-2018 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * BSD-3-Clause license. See the accompanying LICENSE file for details.
+ */
+
+
+#ifndef REALSENSE2_DRIVER_H
+#define REALSENSE2_DRIVER_H
+
+#ifdef WIN32
+#define _USE_MATH_DEFINES
```

Comment:
It is usually safer to add `_USE_MATH_DEFINES` from the CMakeLists.txt file
```cmake
add_definitions(-D_USE_MATH_DEFINES)
```
in order to avoid unwanted interactions among multiple `#include <cmath>`

Btw, you need to include `cmath` within this file anyway to get `M_PI` define.